Modix/Modules/NoteModule.cs

```diff
@@ -0,0 +1,172 @@
+﻿namespace Modix.Modules
+{
+    using System;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Discord;
+    using Discord.Commands;
+    using Discord.WebSocket;
+    using Modix.Data.Models;
+    using Modix.Data.Models.Moderation;
+    using Modix.Services.Moderation;
+    using Serilog;
+    using Services.CommandHelp;
+    using Tababular;
+
+    [Name("Add/Remove/Search Admin notes"), RequireUserPermission(GuildPermission.Administrator), HiddenFromHelp]
+    public class NoteModule : ModuleBase
+    {
+        private readonly IModerationService _moderationService;
+
+        public NoteModule(IModerationService moderationService)
+        {
+            _moderationService = moderationService;
+        }
+
+        [Command("note add"), Summary("Adds a note to a user")]
+        public async Task CreateNote(IGuildUser user, [Remainder] string message)
+        {
+            var guildUser = Context.User as SocketGuildUser;
+
+            // TODO: We shouldn't need to do this once claims are working
+            if (!IsStaff(guildUser) && !IsOperator(guildUser))
+            {
+                await ReplyAsync($"I'm sorry, @{guildUser.Nickname}, I'm afraid I can't do that.");
+                return;
+            }
+
+            try
+            {
+                if (string.IsNullOrWhiteSpace(message))
+                {
+                    Log.Warning($"User: {Context.Message.Author.Username} has tried to create a note without a message");
+                    await ReplyAsync("Please ensure that you have included a message with your note");
+                }
+
+                await _moderationService.CreateInfractionAsync(InfractionType.Notice, guildUser.Id, message, null);
+
+                await ReplyAsync($"Note has been created for user: {guildUser.Username}");
+            }
+            catch (Exception e)
+            {
+                Log.Error(e, $"NoteModule CreateNote failed with the following user: {guildUser.Username} and message: {message}");
+                await ReplyAsync("Uh oh. Looks like you are using the command wrong or an error has occured.");
+            }
+        }
+
+        [Command("note remove"), Summary("Removes a note from a user")]
+        public async Task RemoveNote(long noteId)
```

Comment:
instead of Command("note xxxx"), the better approach is to use the [Group("note")] attribute on the class, and simply the Command attribute to just "xxxx"

Reply:
Fixed in 97fee99.